Index: app/code/Magento/AdminNotification/Block/Window.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/code/Magento/AdminNotification/Block/Window.php	(date 1513000005000)
+++ app/code/Magento/AdminNotification/Block/Window.php	(date 1543933272000)
@@ -14,7 +14,7 @@
 class Window extends \Magento\Backend\Block\Template
 {
     /**
-     * XML path of Severity icons url
+     * XML path of Severity icons url!
      */
     const XML_SEVERITY_ICONS_URL_PATH = 'system/adminnotification/severity_icons_url';
 
